client/blocks/conversation-caterpillar/index.jsx

```diff
@@ -96,6 +104,11 @@ class ConversationCaterpillarComponent extends React.Component {
 		const lastAuthorName = get( last( displayedAuthors ), 'name' );
 		const gravatarSmallScreenThreshold = MAX_GRAVATARS_TO_DISPLAY / 2;
 
+		const hideShowAllHandler = isEmpty( hiddenComments ) ? this.handleHideAll : this.handleShowAll;
+		const hideShowAllText = isEmpty( hiddenComments )
+			? translate( 'Hide all' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **4** times:
`translate( 'Hide All' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).